test/test_utils.py

```diff
@@ -216,7 +225,3 @@ def test_draw_segmentation_masks_errors():
     with pytest.raises(ValueError, match="It seems that you passed a tuple of colors instead of"):
         bad_colors = ('red', 'blue')  # should be a list
         utils.draw_segmentation_masks(image=img, masks=masks, colors=bad_colors)
-
-
-if __name__ == '__main__':
-    unittest.main()
```

Comment:
To preserve the use of `python file.py`, let's change this into:

```py
if __name__ == '__main__':
    pytest.main([__file__])
```

Reply:
Hi @NicolasHug , Done as suggested.